_pytest/terminal.py

```diff
@@ -135,16 +139,20 @@ def __init__(self, config, file=None):
         self.showfspath = self.verbosity >= 0
         self.showlongtestinfo = self.verbosity > 0
         self._numcollected = 0
+        self._session = None
 
         self.stats = {}
         self.startdir = py.path.local()
         if file is None:
             file = sys.stdout
         self._writer = _pytest.config.create_terminal_writer(config, file)
+        self._screen_width = self.writer.fullwidth
         self.currentfspath = None
         self.reportchars = getreportopt(config)
         self.hasmarkup = self.writer.hasmarkup
         self.isatty = file.isatty()
+        self._progress_items_reported = 0
```

Comment:
Do we really have no way already to get how many tests have run so far?

Reply:
Not that I know of; usually this is not really tracked by pytest explicitly, each `runtestprotocol` just triggers the appropriate hooks and each plugin does with that information what it needs.